src/System.Management/src/System.Management.csproj

```diff
@@ -0,0 +1,77 @@
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
+      <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.FreeLibrary.cs">
+      <Link>Common\Interop\Windows\kernel32\Interop.FreeLibrary.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.GetProcAddress.cs">
+      <Link>Common\Interop\Windows\kernel32\Interop.GetProcAddress.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.LoadLibrary.cs">
+      <Link>Common\Interop\Windows\kernel32\Interop.LoadLibrary.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs">
+      <Link>Common\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs</Link>
+    </Compile>
+    <Compile Include="System\Management\ManagementBaseObject.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Include="System\Management\ManagementClass.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Include="System\Management\ManagementDateTime.cs" />
+    <Compile Include="System\Management\ManagementEventArgs.cs" />
+    <Compile Include="System\Management\ManagementEventWatcher.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Include="System\Management\ManagementException.cs" />
+    <Compile Include="System\Management\ManagementNamedValueCollection.cs" />
+    <Compile Include="System\Management\ManagementObject.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Include="System\Management\ManagementObjectCollection.cs" />
+    <Compile Include="System\Management\ManagementObjectSearcher.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Include="System\Management\ManagementOperationWatcher.cs" />
+    <Compile Include="System\Management\ManagementOptions.cs" />
+    <Compile Include="System\Management\ManagementPath.cs" />
+    <Compile Include="System\Management\ManagementQuery.cs" />
+    <Compile Include="System\Management\ManagementScope.cs" />
+    <Compile Include="System\Management\Method.cs" />
+    <Compile Include="System\Management\MethodSet.cs" />
+    <Compile Include="System\Management\Property.cs" />
+    <Compile Include="System\Management\PropertySet.cs" />
+    <Compile Include="System\Management\Qualifier.cs" />
+    <Compile Include="System\Management\QualifierSet.cs" />
+    <Compile Include="System\Management\WmiEventSink.cs" />
+    <Compile Include="System\Management\wmiutil.cs" />
+    <Compile Include="System\Management\WMIGenerator.cs" />
```

Comment:
For some reason I thought @AlexGhiondea found the code generation features could not be ported?

Reply:
There is "instrumentation" (generation of of wmi objects) that can't be done. This is generation of c#, VB code via CodeDom for existent WMI classes. We are good at least for c#, there is a unit test covering that.